src/app/story/directives/status-control.component.ts

```diff
@@ -0,0 +1,357 @@
+import { Component, DoCheck, Input, Output, EventEmitter, ViewChild, ElementRef } from '@angular/core';
+import { interval } from 'rxjs';
+import { skipWhile, mergeMap } from 'rxjs/operators';
+import { StoryModel } from 'app/shared';
+import { ToastrService, ModalService } from 'ngx-prx-styleguide';
+import { Router } from '@angular/router';
+import { Angulartics2 } from 'angulartics2';
+import { StoryStatus } from './status.component';
+
+interface PublishBoxButton {
+  text: string,
+  action: Function,
+  disabled?: boolean,
+  color?: 'red' | 'orange' | 'green' | 'plain',
+  icon?: string
+}
+@Component({
+  selector: 'publish-status-control',
+  template: `
+    <hr>
+    <div class="actions" *ngIf="story">
+      <prx-button [model]="story" working=0 disabled=0 plain=1
+        [visible]="isChanged || nextStatus !== currentStatus" (click)="discard()">Discard</prx-button>
```

Comment:
The alignment on this discard button irks me for some reason.  Would aligning it left help, or would that also be weird?  As in

![image](https://user-images.githubusercontent.com/1410587/59375545-99713280-8d0b-11e9-9a68-df0127352d97.png)

Reply:
Agreed with this one. Lets right align the discard button.